packages/react-router/lib/dom/ssr/single-fetch.tsx

```diff
@@ -657,13 +677,13 @@ function unwrapSingleFetchResult(result: SingleFetchResult, routeId: string) {
   } else if ("redirect" in result) {
     let headers: Record<string, string> = {};
     if (result.revalidate) {
-      headers["X-Remix-Revalidate"] = "yes";
+      headers["X-Remix-Revalidate"] = "true";
     }
     if (result.reload) {
-      headers["X-Remix-Reload-Document"] = "yes";
+      headers["X-Remix-Reload-Document"] = "true";
     }
     if (result.replace) {
-      headers["X-Remix-Replace"] = "yes";
+      headers["X-Remix-Replace"] = "true";
```

Comment:
Align these with values used elsewhere - value doesn't really matter since we use `.has(header)`